src/app/ASKS/ask/index.js

```diff
@@ -695,21 +697,25 @@ const Ask = ({
           clientId={clientId}
         />
       )}
-      {component === 'flag' && (
-        <Flag.Write
-          data={data}
-          questionCode={questionCode}
-          onSendAnswer={onSendAnswer}
-          regexPattern={regexPattern}
-          errorMessage={errorMessage}
-          parentCode={parentCode}
-          placeholderName={placeholderName}
-          attributeCode={attributeCode}
-          targetCode={targetCode}
-          clientId={clientId}
-          mandatory={mandatory}
-        />
-      )}
+      {component === 'flag' ||
```

Comment:
Instead of 
` 
{component === 'flag' ||
        (component === 'labelled_flag' && (
        ....
 `
    It should be:
 `
{(component === 'flag' ||
        component === 'labelled_flag') && (
        ...
`    
**Error**:       The first one that you have means that either the component needs to equal to 'flag' or component needs to equal to 'labelled_flag' while also <Flag.Write component> needs to resolve to true. And there is no return component.
        
 **Fix**:    The second approach means the component needs to equal to 'flag' or the component needs to equal to 'labelled_flag'. Whenever either one of these are true return the <Flag.Write /> component